framework/src/media/MediaPlayerImpl.cpp

```diff
@@ -795,7 +795,14 @@ void MediaPlayerImpl::playback()
 			}
```

Comment:
It's not directly related to this commit, but how about moving line 789 and 790 into this position and removing line 793 and 794 for the sake of simplicity?

Reply:
I'll fix in another PR if needed